src/app/page.js

```diff
@@ -1,8 +1,21 @@
 import Landing from "@/components/home/Landing";
-
+import Event from "@/components/events/Event";
 const home = () => {
   return (
     <div>
+      <div className="relative py-3 sm:max-w-xl sm:mx-auto">
```

Comment:
it's ok to add the component to the page for seeing what it looks like but make sure you remove it before you push